### PR TITLE
Fixes #12231 - Allow editing smart variables with periods in puppetcl…

### PR DESCRIPTION
**Pull request by tbrisker**
…ass form

### CODE REVIEW

File: app/helpers/lookup_keys_helper.rb [hunk 1]
```diff
@@ -36,9 +36,7 @@ def show_puppet_class(f)
     if params["puppetclass_id"]
       select_f f, :puppetclass_id, [Puppetclass.find(params["puppetclass_id"])], :id, :to_label, {}, {:label => _("Puppet class"), :disabled => true}
     elsif f.object.puppet? && f.object.param_class
-      field(f, :puppetclass_id, :label => _('Puppet Class')) do
-        content_tag(:input, nil, :value => f.object.param_class, :type => 'text', :disabled => true)
-      end
+      text_f(f, :puppetclass_id, :label => _('Puppet Class'), :value => f.object.param_class, :disabled => true)
```
Comment (dLobatog) [inline app/helpers/lookup_keys_helper.rb:39]: Boyscout rule :smile: ?

**Comment by dLobatog** [comment]
Merged as 70417537f362acd77802340a0de5a0e5d3f6cf5c, thanks @tbrisker!

**Comment by dLobatog** [comment]
51c05864e034d229efe4a4673ec85189da60e90c, sorry

File: app/helpers/lookup_keys_helper.rb (hunk 1 above)
Comment (tbrisker) [inline app/helpers/lookup_keys_helper.rb:39]: :+1: When testing I noticed that specific field looked odd, so fixed it on the way.

**Pull request closed without merging**